### ISSUE TITLE
Date Picker a11y bug - Not announcing the day for a date while navigating in the calendar

### ISSUE THREAD

**Issue by niketk** [OfficeDev/office-ui-fabric-react#5339]
### Bug Report 
 
- __Package version(s)__: 5.95.0
- __Browser and OS versions__: Chrome and Win 10
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) No
 
Requested priority: (Blocking, High, Normal, Low)  High
 
Products/sites affected: (if applicable) 
 
#### Describe the issue:
 
Not announcing the day for a date while navigating in the calendar
 
#### Actual behavior:
 
Screen reader will announce date , month year but not the day.
 
#### Expected behavior:
 
After selecting a date, narrator does announce the DAY for the selected day as well. But also, while navigating in the calendar, DAY should be announced because every time it will be difficult for a user to select random dates to know the DAY for that day. 
 
MAS Reference : MAS 4.2.1 - Object Information (33A), MAS 4.2.3 - Row, Column, and Headers (33C), MAS 4.2.6 - Parent-Child Relationships (33F)

**Comment by micahgodbolt** [comment]
By day, you mean day of the week, correct?

**Comment by micahgodbolt** [comment]
Reopening to give the OP a chance to reply.
